spec/models/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'Testing for no Title' do
+    post = Post.create
+    expect(post).to_not be_valid
+  end
+
+  it 'Testing post CommentCounter number negative' do
+    post = Post.create(AuthorId: 1, Title: 'John', Text: 'anything')
+    post.CommentCounter = -10
+    expect(post).to_not be_valid
+  end
+
+  it 'Testing post LikeCounter number negative' do
+    post = Post.create(AuthorId: 1, Title: 'John', Text: 'anything')
+    post.LikeCounter = -10
+    expect(post).to_not be_valid
+  end
+
+  it 'Title should be present' do
+    post = Post.create(AuthorId: 1, Title: 'John', Text: 'anything')
+    post.Title = nil
+    expect(post).to_not be_valid
+  end
+
+  it 'Title should not exceed 250 characters' do
+    subject.Title = 'a' * 300
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] As requested by the previous reviewer, Please add a test for the custom method available in the Post model `latest_comments`. This test should check 5 recent comments for a post.